### PR TITLE
Optout validates JWT

### PR DESCRIPTION
**Pull request by thomasm-ttd**

### CODE REVIEW

File: src/main/java/com/uid2/optout/Main.java [hunk 1]
```diff
@@ -116,8 +118,12 @@ public Main(Vertx vertx, JsonObject config) throws Exception {
         final DownloadCloudStorage contentStorage;
         if (coreAttestUrl != null) {
             String coreApiToken = this.config.getString(Const.Config.CoreApiTokenProp);
+            if (coreApiToken == null || coreApiToken.isBlank()) {
+                LOGGER.warn("Core Attest URL set, but the API Token is not set");
```
Comment (aulme) [inline src/main/java/com/uid2/optout/Main.java:122]: What does this mean? Is this an error case?

File: src/main/java/com/uid2/optout/vertx/OptOutServiceVerticle.java [hunk 2]
```diff
@@ -161,7 +171,7 @@ private Router createRouter() {
         router.route(REPLICATE_METHOD)
                 .handler(auth.handle(this::handleReplicate, Role.OPTOUT));
         router.route(REFRESH_METHOD)
-                .handler(auth.handle(attest.handle(this::handleRefresh), Role.OPERATOR));
+                .handler(auth.handle(attest.handle(this::handleRefresh, Role.OPERATOR), Role.OPERATOR));
```
Comment (aulme) [inline src/main/java/com/uid2/optout/vertx/OptOutServiceVerticle.java:174]: Is this really supposed to be the Operator's role? Not Optout?

File: conf/local-config.json [hunk 3]
```diff
@@ -12,5 +12,9 @@
   "partners_metadata_path": "/com.uid2.core/test/partners/metadata.json",
   "clients_metadata_path": "/com.uid2.core/test/clients/metadata.json",
   "operators_metadata_path": "/com.uid2.core/test/operators/metadata.json",
-  "optout_observe_only": false
+  "optout_observe_only": false,
+  "aws_kms_jwt_signing_public_keys": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9tXqXNeeuA3lmfNF2AF06OLY6icxEOlXaZEXyh62ieFjakvDeR438s5gA1ToKqpW5uTPkKtP1YjHf16ofjlDYcLxCKOombjW1m2t/3FFfWRaZgr/SFr5TaI3N86tAvf313kql8GVrre/HQLhaHUXQJxDYh3/Wl7+KSMrMwMQCwkgnf1AmGUcPl1AzxuKUia8IM9joZKq9UpxRdWkLeqJ47d0bX5+SFLOwRhFxnejNx3iJYpEi43aFxLnr1WmTtMrIkHYmA2GENnhXsrV3n0bqLUah+HS6S+m3lBUW09TEFbagtktCtsFqvE+YExXo8b+hOnlakQtsbtJY0xg8Z1AtwIDAQAB,MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9tXqXNeeuA3lmfNF2AF06OLY6icxEOlXaZEXyh62ieFjakvDeR438s5gA1ToKqpW5uTPkKtP1YjHf16ofjlDYcLxCKOombjW1m2t/3FFfWRaZgr/SFr5TaI3N86tAvf313kql8GVrre/HQLhaHUXQJxDYh3/Wl7+KSMrMwMQCwkgnf1AmGUcPl1AzxuKUia8IM9joZKq9UpxRdWkLeqJ47d0bX5+SFLOwRhFxnejNx3iJYpEi43aFxLnr1WmTtMrIkHYmA2GENnhXsrV3n0bqLUah+HS6S+m3lBUW09TEFbagtktCtsFqvE+YExXo8b+hOnlakQtsbtJY0xg8Z1AtwIDAQAB",
```
Comment (aulme) [inline conf/local-config.json:16]: I see it says public key, is it ok to be checked in? Or is this a fictitious test key?

File: src/main/java/com/uid2/optout/Main.java (hunk 1 above)
Comment (thomasm-ttd) [reply src/main/java/com/uid2/optout/Main.java:122]: This is to try and help figure out why attestation is failing. I want to see if the key is set

File: src/main/java/com/uid2/optout/vertx/OptOutServiceVerticle.java (hunk 2 above)
Comment (thomasm-ttd) [reply src/main/java/com/uid2/optout/vertx/OptOutServiceVerticle.java:174]: Yes, it really is. The operator calls this method. You can see the auth handler has the same role

File: conf/local-config.json (hunk 3 above)
Comment (thomasm-ttd) [reply conf/local-config.json:16]: Public keys are not sensitive, so it can be checked in. It is a key we can use in e2e testing.

**Review by aulme — APPROVED** [review]

**Pull request merged**